test/framework/src/main/java/org/elasticsearch/node/MockNode.java

```diff
@@ -134,6 +138,14 @@ protected SearchService newSearchService(ClusterService clusterService, IndicesS
             bigArrays, fetchPhase, circuitBreakerService);
     }
 
+    @Override
+    protected ScriptService newScriptService(Settings settings, Map<String, ScriptEngine> engines, Map<String, ScriptContext<?>> contexts) {
+        if (getPluginsService().filterPlugins(MockScriptService.TestPlugin.class).isEmpty()) {
```

Comment:
We need to actually add this test plugin to the mock plugins. See `ESIntegTestCase.getMockPlugins()` and ESSingleNodeTestCase.newNode(). For now we can probably just always add it.

Reply:
Added.